shelley/chain-and-ledger/formal-spec/update.tex

```diff
@@ -1,26 +1,140 @@
-\section{Updates}
+\section{Update Proposal Mechanism}
 \label{sec:update}
 
 
 The $\mathsf{UPDATE}$ transition is responsible for the federated governance model in Shelley.
-This includes the ability to change the protocol parameters and also handles software
-updates.
-
-The application versions behave much like a protocol parameter, but because adoption
-for them must occur as soon as consensus is met
-\footnote{as opposed to waiting until the end of an epoch},
-they are handled separately.
-The adoption of new application versions is what triggers software updates.
-Therefore $\mathsf{UPDATE}$ is the combination of the
-$\mathsf{PPUP}$ transition for the protocol parameters and the
-the $\mathsf{AVUP}$ transition for the application versions.
-Note that $\PPUpdate$, $\AVUpdate$, and $\Update$ were defined in
-Figure~\ref{fig:defs:utxo-shelley}, and that $\fun{pvCanFollow}$
-was defined in \cite{byron_ledger_spec}.
-
+The governance process includes a mechanism for core nodes to propose and vote on
+updates. In this chapter we
+outline rules for genesis keys \textit{proposing} both protocol parameter
+and application version updates, as well as voting on whether a particular
+software update is an acceptable future option (again, only genesis keys vote on this).
+For rules regarding the \textit{adoption} of protocol
+parameter updates, see \ref{sec:pparam-update}. For rules regarding
+adoption of new software versions see \ref{sec:software-updates}. \newline
+
+\noindent \underline{Distinction between protocol parameter and application updates.}
+\begin{itemize}
+\item \textbf{Protocol parameters} (excluding $\ProtVer$, explained below): constants
+currently used in ledger calculations according to the rules described in this document
+\begin{itemize}
+\item[$\circ$] Stored on the leger
+\item[$\circ$] Changed only at epoch boundary
+\item[$\circ$] Changed according to the genesis key votes, which are also stored on the ledger
+\item[$\circ$] All nodes automatically adopt new values (this mechanism is an explicit
+ part of the ledger rules, and is included in the epoch boundary transition)
+\item[$\circ$] Has no effect on ledger transition rules
```

Comment:
If we change, say, the max transaction size, then in a sense there is an effect on the transition rules. I guess you mean some kind of logic change?